doc/root.markdown.in

```diff
@@ -6,7 +6,6 @@ OpenCV modules {#mainpage}
 - @ref tutorial_py_root
 @CMAKE_DOXYGEN_TUTORIAL_JS_ROOT@
 @CMAKE_DOXYGEN_TUTORIAL_CONTRIB_ROOT@
-- @ref faq
```

Comment:
For better experience:
1. We should keep FAQ page in documentation (compatibility).
2. Replace current FAQ page content with the **link** on new Wiki page.
3. This link may forward to Wiki page directly.

Reply:
Added direct link to Wiki in markdown format.

Reply:
I think it is more important to keep the FAQ page itself to keep links `/d3/d2d/faq.html` working. This page will have a message: _FAQ page have been moved to GitHub_ 

Otherwise drop-down menu can lead users to 404 page.

Reply:
@alalek I replaced markdown notation with HTML and checked documentation with `linkchecker` tool. There are nothing new with FAQ page there. Do you mean version selector in documentation header?

Reply:
Yes, this one:
![image](https://user-images.githubusercontent.com/3304494/78722206-ad88bf00-7931-11ea-97f6-33211abb633c.png)

Maybe it is not very frequent scenario, but it is possible and I stumble upon it sometimes on https://docs.python.org/ and on https://cmake.org/documentation. When web search leads you to some page for version X and you want to switch to more recent version Y but this page is moved or deleted and you get 404.

Reply:
@alalek What do you think? I suppose that it's normal situation, because project is alive, new pages appear and disapear because of refactoring. The issue become non relevant for most of users with the next release.

Reply:
> 1. We should keep FAQ page in documentation (compatibility).

Here.

Reply:
Ok, replaced the page content with compatibility message and link.